src/credentials/examples/DeviceAttestationVerifierExample.cpp

```diff
@@ -231,20 +217,20 @@ AttestationVerificationResult ExampleDACVerifier::VerifyAttestationInformation(c
     // match DAC and PAI VIDs
     if (!paiCertDerBuffer.empty())
     {
-        VendorId paiVid;
-        VendorId dacVid;
+        uint16_t paiVid;
+        uint16_t dacVid;
```

Comment:
Please set both of them to 0 before starting, to guard in case of missed error cases

Reply:
Fixed in 9cddb31